### PR TITLE
Editor unit tests and minor fixes

### PR DESCRIPTION
**Pull request by taki9**

### CODE REVIEW

File: packages/sn-editor-react/test/editor.test.tsx [hunk 1]
```diff
@@ -0,0 +1,48 @@
+import { EditorContent } from '@tiptap/react'
+import { mount, shallow } from 'enzyme'
+import React from 'react'
+import { act } from 'react-dom/test-utils'
+import { Editor } from '../src/components/editor'
+import { MenuBar } from '../src/components/menu-bar'
+
+describe('editor', () => {
+  it('should render properly', () => {
```
Comment (herflis) [inline packages/sn-editor-react/test/editor.test.tsx:9]:
```suggestion
  it('should be rendered properly', () => {
```

File: packages/sn-editor-react/test/editor.test.tsx [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+    const value = '<p>Hello World</p>'
+    const wrapper = mount(<Editor content={value} autofocus={true} readOnly={true} />)
+
+    const { options } = wrapper.find(EditorContent).prop('editor')
+    expect(options.content).toBe(value)
+    expect(options.editable).toBe(false)
+    expect(options.autofocus).toBe(true)
+  })
+
+  it('should render placeholder if no content passed', () => {
```
Comment (herflis) [inline packages/sn-editor-react/test/editor.test.tsx:25]:
```suggestion
  it('should render placeholder if no content is passed', () => {
```

File: packages/sn-editor-react/test/image-control.test.tsx [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+  const editor = new TiptapEditor({
+    content: '<p>Hello world</p>',
+    extensions: createExtensions(),
+    onUpdate: onChange,
+  })
+
+  const fileReader = new FileReaderMock()
+  jest.spyOn(window, 'FileReader').mockImplementation(() => fileReader)
+
+  it('should render properly', () => {
```
Comment (herflis) [inline packages/sn-editor-react/test/image-control.test.tsx:21]:
```suggestion
  it('should be rendered properly', () => {
```

File: packages/sn-editor-react/test/link-control.test.tsx [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+
+describe('link control', () => {
+  const onChange = jest.fn(({ editor }) => editor.getHTML())
+  const editor = new TiptapEditor({
+    content: '<p>Hello world</p>',
+    extensions: createExtensions(),
+    onUpdate: onChange,
+  })
+
+  it('should render properly', () => {
```
Comment (herflis) [inline packages/sn-editor-react/test/link-control.test.tsx:17]:
```suggestion
  it('should be rendered properly', () => {
```

File: packages/sn-editor-react/test/link-control.test.tsx [hunk 5]
```diff
@@ -0,0 +1,73 @@
...
+    editor.commands.setTextSelection({ from: 0, to: 6 })
+
+    const wrapper = shallow(<LinkControl editor={editor} />)
+    expect(wrapper.find(Dialog).prop('open')).toBe(false)
+    wrapper.find(IconButton).simulate('click')
+    expect(wrapper.find(Dialog).prop('open')).toBe(true)
+    expect(wrapper.find(DialogTitle).text()).toBe(defaultLocalization.linkControl.title)
+  })
+
+  it('should not open dialog when no text selected', () => {
```
Comment (herflis) [inline packages/sn-editor-react/test/link-control.test.tsx:33]:
```suggestion
  it('should not open dialog when no text is selected', () => {
```

File: packages/sn-editor-react/test/typography-control.test.tsx [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+import { defaultLocalization } from '../src/context'
+import { createExtensions } from '../src/extension-list'
+
+describe('typography control', () => {
+  const editor = new TiptapEditor({
+    content: '<h6>Hello</h6><p>world</p>',
+    extensions: createExtensions(),
+  })
+
+  it('should render properly', () => {
```
Comment (herflis) [inline packages/sn-editor-react/test/typography-control.test.tsx:15]:
```suggestion
  it('should be rendered properly', () => {
```

**Review by herflis — CHANGES_REQUESTED** [review]

**Review by pusztaienike — APPROVED** [review]
Don't forget the conflicts!

**Pull request merged**